Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Indexer/NodeIndexer.php

```diff
@@ -110,6 +110,18 @@ public function getIndex() {
 	}
 
 	/**
+	 * Something like getContextPath() but using the Node Identifier instead of the Path
+	 *
+	 * @param NodeInterface $node
+	 * @return string
+	 */
+	protected function getContextIdentifier(NodeInterface $node) {
+		$contextPath = $node->getContextPath();
+
+		return $node->getIdentifier() . substr($contextPath, strpos($contextPath, '@'));
```

Comment:
The use of substr() should be replaced by using getWorkspace()->getName() instead, analogous to getContextPath().